tests/helpers/server.ts

```diff
@@ -0,0 +1,67 @@
+import { Locator, Page } from "@playwright/test";
+import { TIMEOUT } from "./constants";
+
+export type ServerSettingType = "checkbox" | "text" | "number" | "password";
+
+export const visitServerSettings = async (page: Page) => {
+  await page.goto("/ui/");
+  await page.getByRole("link", { name: "Settings", exact: true }).click();
+};
+
+export const activateSetting = async (page: Page, settingName: string) => {
+  await page
+    .locator("css=tr", { hasText: settingName })
+    .getByRole("button")
+    .click();
+};
+
+export const updateCheckbox = async (settingRow: Locator) => {
+  const checkbox = settingRow.locator(".p-checkbox__label");
+  await checkbox.click();
+};
+
+export const checkIfChangeButtonExist = async (settingRow: Locator) => {
+  const changeButton = settingRow.getByRole("button", {
+    name: "Change",
+    exact: true,
+  });
+  return changeButton.isVisible({ timeout: 1000 });
+};
+
+export const updateSetting = async (
+  page: Page,
+  settingName: string,
+  settingType: ServerSettingType,
+  content?: string,
+) => {
+  const settingRow = page.locator("css=tr", { hasText: settingName });
+  if (settingType === "checkbox") {
+    await updateCheckbox(settingRow);
+  } else {
+    const settingInput = settingRow.locator(`css=input[type=${settingType}]`);
+    if (content) {
+      await settingInput.fill(content);
+    }
+  }
+  await settingRow.getByRole("button", { name: "Save", exact: true }).click();
+  await page.waitForSelector("text=Setting updated.", TIMEOUT);
```

Comment:
I think we should close the notification, so the next check doesn't get marked as success by the notification from the previous setting that ran before. 

Maybe also validate the value if possible? I.e. true/false for checkbox, `value` for string and `set` for passwords?

Reply:
agreed, closing the notification for both update and reset. Also adding validations for both updates and resets.